skyportal/tests/api/test_annotation_services.py

```diff
@@ -51,3 +51,44 @@ def test_irsa_wise(public_group, upload_data_token):
             for d in data['data']
         ]
     )
+
+
+@pytest.mark.flaky(reruns=2)
+def test_vizier_quasar(
+    annotation_token, public_source, public_group, upload_data_token
```

Comment:
Don't need `public_source`.

Reply:
and also maybe not need `annotation_token`.

Reply:
Indeed.